extra/src/main/java/aima/extra/probability/ProbabilityNumber.java

```diff
@@ -146,4 +171,12 @@
 	 * @return true if the probabilities sum to one, false otherwise.
 	 */
 	boolean sumsToOne(Iterable<ProbabilityNumber> allProbabilities);
+
+	/**
+	 * Override the precision of ProbabilityNumber instances returned as a
+	 * result of performing operations.
+	 * 
+	 * @param mc
+	 */
+	void overrideComputationPrecisionGlobally(MathContext mc);
```

Comment:
I think this might better belong in the instantiation mechanism exclusively. Not sure.